youths/schema.py

```diff
@@ -0,0 +1,90 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id"]
+
+
+PreferredLanguage = graphene.Enum(
+    "PreferredLanguage", [(l[0].upper(), l[0]) for l in LANGUAGES]
+)
+Gender = graphene.Enum("Gender", [(g[0].upper(), g[0]) for g in GENDERS])
```

Comment:
indexes are out of sync here as well 🙂

Reply:
Forgot to discuss this with you. It does not work if the label is set to 1 as there is non ascii caracters.

Reply:
ah, ok, the name of the enum is checked against a regex by graphene and apostrophes are apparently not allowed. 😅 let's keep it as is here.